src/qml/qgismobileapp.qml

```diff
@@ -805,6 +842,25 @@ ApplicationWindow {
       width: 5
     }
 
+    /* Highlight the currently selected item being rotated */
+    FeatureListSelectionHighlight {
```

Comment:
Could we instead use only a single FeatureListSelectionHighlight for both moving and rotating features? Thanks.